app/services/teacher_training_api/create_course.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module TeacherTrainingApi
+  class CreateCourse
```

Comment:
Should we call this `ImportCourse` or something like that? I thought this was writing to the API at first.

Reply:
Agreed, I prefer `ImportCourse`

Reply:
Should I also rename the corresponding jobs? e.g. `TeacherTrainingApi::ImportCoursesJob`

Reply:
I renamed them all for consistency 👍